src/groups/group.jl

```diff
@@ -913,6 +938,46 @@ end
     X,
     conv::ActionDirectionAndSide=LeftForwardAction(),
 )
+# the following are true if the tangent vectors are stored with the left invariant convention
```

Comment:
Same question: Do we assume this implicitly or make that sure somehow?

Reply:
Same here. The purpose of those comments is to emphasise precisely which functions depend on that storage convention.

Reply:
Done: put in the docstring.